spec/support/edit_fields/edit_field.rb

```diff
@@ -10,17 +10,35 @@ class EditField
 
   attr_accessor :field_type
 
+  # Initializes a new EditField object. It represents a work package field on a
+  # work packages table page, a work package view page (split or full), or a
+  # work package creation page (split or full).
+  #
+  # @param context [Object] The context in which the EditField is being used.
+  # @param property_name [Symbol] The name of the property associated with the
+  #   EditField. Generally camel case.
+  # @param selector [String] (optional) The CSS selector used to locate the
+  #   EditField element. if unspecified, the `property_name` is used.
+  # @param create_form [Boolean] (optional) Indicates whether the EditField is
+  #   used in a create form. It changes the way the field is clicked or not to
+  #   activate it on edition. It is `false` by default.
```

Comment:
This is extremely valuable to have handy, thanks.

Reply:
Yeah, most of it was generated by GitHub Copilot actually. I reviewed it and updated it where it made sense. It saves a lot of time about looking up what the exact RDoc syntax is.

All the doc added in `spec/support/pages/work_packages/work_packages_table.rb` has been generated this way.